src/js/components/modals/AppModalComponent.jsx

```diff
@@ -49,7 +49,8 @@ var AppModalComponent = React.createClass({
     return {
       fields: AppFormStore.fields,
       errorIndices: {},
-      responseErrorMessages: {}
+      responseErrorMessages: {},
+      force: false
```

Comment:
Actually I like this approach.
With settings the state, you could also change the title of the submit button later on.